### ISSUE TITLE
APA Personalization TF I18N Self Review: Adaptable Content Module

### ISSUE THREAD

**Issue by clapierre** [w3c/i18n-activity#862]
Here is a link to our explainer document which may help for context.
[Personalization Semantics Explainer 1.0](https://www.w3.org/TR/personalization-semantics-1.0/)

[Personalization Semantics Content Module 1.0](https://www.w3.org/TR/personalization-semantics-content-1.0/)

 If the spec (or its implementation) contains any natural language text that will be read by a human (this includes error messages or other UI text, JSON strings, etc, etc),
Answer: N/A.

  If the spec (or its implementation) allows content authors to produce typographically appealing text, either in its own right, or in association with graphics.

Answer: We are using numerical tokens which can be mapped to symbols sets, so we don't think this is an issue but request you look at this closer. Also the symbol sets themselves may be localized.
[Spec symbol](https://www.w3.org/TR/personalization-semantics-content-1.0/#symbol-explanation)

  If the spec (or its implementation) allows the user to point into text, creates text fragments, concatenates text, allows the user to select or step through text (using a cursor or other methods), etc.

Answer: Simplification allows for replacing sections of text with an alternative text, which the author will provide.
[Simplification](https://www.w3.org/TR/personalization-semantics-content-1.0/#simplification-explanation)

  If the spec (or its implementation) allows searching or matching of text, including syntax and identifiers
Answer: N/A

  If the spec (or its implementation) sorts text
Answer: N/A

  If the spec (or its implementation) captures user input
Answer: N/A

  If the spec (or its implementation) deals with time in any way that will be read by humans and/or crosses time zone boundaries
Answer: N/A

  If the spec (or its implementation) allows any character encoding other than UTF-8.

Answer: We support UTF-8 but we anticipates supporting UTF-16 for symbol sets and there is nothing in our spec that would prevent this.

  If the spec (or its implementation) defines markup.

Answer:We do define attribute names and values as tokens that are human readable but we don't expect those to be localized. However we do have content simplification that does have alternative text descriptions.
[Simplification](https://www.w3.org/TR/personalization-semantics-content-1.0/#simplification-explanation)

  If the spec (or its implementation) deals with names, addresses, time & date formats, etc
Answer:data-purpose attribute does pattern after autocomplete attribute of HTML5 and author is responsible for any internationalization concerns.

  If the spec (or its implementation) describes a format or data that is likely to need localization.

Answer: data-purpose attribute does pattern after autocomplete attribute of HTML5 and author is responsible for any internationalization concerns.

  If the spec (or its implementation) makes any reference to or relies on any cultural norms

Answer: there are internationalized AAC symbol sets to address that concern. and we are supporting those translations

---
**WHEN CREATING A NEW ISSUE DO SO ABOVE THIS PARAGRAPH, REPLACING THE PROMPTS, BUT LEAVE THIS PARAGRAPH INTACT AS WELL AS THE TEXT BELOW IT** When this issue is raised in the github/bugzilla/mail of the WG that owns the spec, use the text above this para as the basis for that comment. Then edit this issue to remove this paragraph and ALL THE TEXT ABOVE IT. Replace the text 'link_to_issue_raised' below with a link to the place you raised the issue, but leave the remaining text below this para unaltered.

**This is a tracker issue.** Only discuss things here if they are i18n WG internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ link_to_issue_raised

**Comment by r12a** [comment]
@clapierre thanks for doing this review, but only i18n WG folks should raise issues in the i18n-activity repo.  Could you copy the data above back to your own repo?  Besides the fact that issues in this repo are for tracking only, you'll find that others in your group will not be notified of any discussion that ensues here (such as this comment, for example).

When you have moved the content to your repo i'll delete this thread.

thanks

**Comment by clapierre** [comment]
Sorry got that URI to the GitHub from 
https://w3c.github.io/i18n-drafts/techniques/shortchecklist

The text of this page is still being perfected, and is not exhaustive. Please raise any comments using [GitHub issues](https://github.com/w3c/i18n-activity/issues/).

and clicked on that link for GitHub Issues.
May want to make it super clear that is for internal use, and for folks that are requesting a review to click on the 
Request an i18n review l
which is off to the right side which I didn't notice.
BTW I am low vision and wasn't expecting to see that over on the right hand side.

I might suggest you make it super obvious on which link to use

I will now move this request over to that 
https://github.com/w3c/i18n-request/
repository.
Thanks

**Comment by r12a** [comment]
Thanks for revealing several holes in our process!  I have made edits to https://w3c.github.io/i18n-drafts/techniques/shortchecklist and https://www.w3.org/International/review-request#selfreview to make things clearer in future.

I have also created an issue template in i18n-request repo, which should help too.

I then raised a new issue https://github.com/w3c/i18n-request/issues/110 following the new template, and pointed back to your original location for the self-review report at https://github.com/w3c/personalization-semantics/issues/133#issuecomment-593450546.  This means that any discussion around the review results will take place in your repo, and will be seen by your WG.

Thanks for helping us make progress here. I'll now close this issue, since it is redundant, but feel free to add further comments here related to the process or the changes i made.
